### ISSUE TITLE
Running tsc-fancy results in `env: node\r: No such file or directory`

### ISSUE THREAD

**Issue by TimonVS** [JoshuaKGoldberg/tsc-fancy#1]
Running tac-fancy results in the following error: `env: node\r: No such file or directory`

<img width="752" alt="terminal" src="https://user-images.githubusercontent.com/876666/36377960-3d111330-1579-11e8-9104-affb8be6b14e.png">

I've created a repo so that the bug can easily be reproduced: https://github.com/TimonVS/tsc-fancy-bug

**Comment by JoshuaKGoldberg** [comment]
Thanks for posting @TimonVS! 

It looks like this is a dup of https://github.com/automutate/autolesshint/issues/61, which reinforces my suspicion that I'm setting up these Node CLIs the wrong way for Mac/Linux. 😢 

I don't normally have either of those with me, but coincidentally am borrowing a Mac this weekend. I'll try to fix both these issues today. 

Edit: What specific OS, shell (zsh?), yarn, npm, and Node versions do you have installed?

**Comment by JoshuaKGoldberg** [comment]
😦 I can't repro this on OS X Yosemite (10.10.5) with 

```shell
MacBook[10]% yarn tsc
yarn run v1.3.2
$ /Users/mark/Desktop/JoshTemp/tsc-fancy-bug/node_modules/.bin/tsc
✨ Done in 1.88s.
```

```shell
MacBook[11]% yarn tsc-fancy
yarn run v1.3.2
/Users/mark/Desktop/JoshTemp/tsc-fancy-bug/node_modules/.bin/tsc-fancy
✨ Done in 2.06s.
```

**Comment by danstiner** [comment]
This is the classic Windows vs Unix line endings (The '\r' is a dead giveaway). You must have published from Windows with a pre-v5.4.0 version (Aug 22, 2017) of npm.

Just re-publish a new version with a newer version of npm and the problem will go away.

issue from npm discussing the fix: https://github.com/npm/npm/issues/12371

This reproduced for me on Linux, and I'd expect it to reproduce on Mac OS as well, are you maybe using yarn link? Thanks to `core.autocrlf`, the git repository has '\n' has line endings, so if you checkout on a Unix platform everything works as expected, it's only because you published from a Windows platform this crops up.

Do I get a cookie? :cookie:

**Comment by JoshuaKGoldberg** [comment]
Released **v1.1.1** with this update. Thanks @danstiner! You get _all_ the cookies.

**Comment by esprehn** [comment]
We're still seeing this bug in v1.1.1, looks like the tsc-fancy script still contains the \r

```
env: node\r: No such file or directory
```

**Comment by JoshuaKGoldberg** [comment]
Sorry @esprehn - I manually removed the `\r`s and published **v1.1.2**. That should work.
